packages/lesswrong/lib/collections/sequences/schema.ts

```diff
@@ -15,89 +32,61 @@ const schema: SchemaType<"Sequences"> = {
     canRead: ['guests'],
     canCreate: ['admins'],
     canUpdate: ['admins'],
+    group: formGroups.adminOptions,
     control: 'FormUserSelect',
-    tooltip: 'The user id of the author',
+    form: {
+      label: "Set author",
+    },
   },
 
   title: {
     type: String,
     optional: false,
     canRead: ['guests'],
-    canUpdate: ['members'],
+    canUpdate: [userOwns, 'admins', 'sunshineRegiment'],
```

Comment:
I also updated some of these permissions, because I don't think all these fields should be editable by any user?